README.md

```diff
@@ -208,8 +208,9 @@ for t in range(500):
   print(t, loss.data[0])
   
   # Manually zero the gradients before running the backward pass
-  w1.grad.data.zero_()
-  w2.grad.data.zero_()
+  if t:
```

Comment:
In the pytorch.org tutorial, `*zero_()` is called after `backward()`. But in this example, w.grade.Data can be None.